docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+
+services:
+
+  meetpy:
+    image: meetpy
+    build: .
+    volumes:
+      - ./:/usr/src/app
+    working_dir: /usr/src/app/meetpy
+    ports:
+      - "8080:8000"
+    tty: true
+    depends_on:
+      - db
+      - smtp
+    environment:
```

Comment:
How about using `env_file` for keeping these? They may be reused later in the `smtp` service as well.

Reply:
Good idea! I'll do it in another PR